server/database/commands.go

```diff
@@ -0,0 +1,64 @@
+package database
+
+import (
+	"database/sql"
+
+	"github.com/rs/zerolog/log"
+)
+
+func InsertMoveCommand(db *sql.DB, id int64, commandType string, target int64) error {
+	log.Info().Msgf("insert into t_command (ship, type, target) values (%d, %s, %d)", id, commandType, target)
```

Comment:
if we want to report called SQL (which I quite like), we can create a generic function for it and wrap the command with it. This is error prone

Reply:
I would leave that for another PR. Creating an issue for that.